edn/src/symbols.rs

```diff
@@ -277,3 +277,30 @@ impl Display for NamespacedKeyword {
         write!(f, ":{}/{}", self.namespace, self.name)
     }
 }
+
+#[macro_export]
+macro_rules! ns_keyword {
+    ($name: expr) => {{
+        let splits = [':', '/'];
+        let mut i = $name.split(&splits[..]);
```

Comment:
This stuff happens at runtime. I suspect you want to just do `ns_keyword { ($ns: expr, $name: expr) …` and make the callers more efficient!